modules/nw-rfhe-creating-hardware-event-kind-cr.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+:_content-type: PROCEDURE
+[id="nw-rfhe-hardware-event-kind-cr_{context}"]
+= Creating the HardwareEvent resource
+
+To start using Baremetal Hardware Event proxy, install the operator and then create the following HardwareEvent.
+
+
+[NOTE]
+====
+You can only create one instance of HardwareEvent in this release.
+====
+
+[source,yaml]
+----
+    apiVersion: "event.redhat-cne.org/v1alpha1"
+    kind: "HardwareEvent"
+    metadata:
+      name: "hardware-event"
+    spec:
+     nodeSelector: {}
+     transportHost: "amqp://amq-router-service-name.amq-namespace.svc.cluster.local" <1>
+----
+
+<1> The `amqp` host that is specified delivers the events at the transport layer using the AMQP protocol.
+
+This operator needs a secret to be created to access Redfish Message Registry. The secret name and spec the operand expects are under the same namespace the operand is deployed on. For example:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: redfish-basic-auth
+type: Opaque
+data:
+  username: cm9vdA==
+  password: Y2Fsdmlu
+stringData:
+  # BMC host DNS or IP address
+  hostaddr: 123.123.123.123
+
+----
+
+Here is an example of updating nodeSelector to select a specific node:
+
+
+[source,yaml]
+----
+apiVersion: "event.redhat-cne.org/v1alpha1"
+kind: "HardwareEvent"
+metadata:
+  name: "hardware-event"
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/hw-event: ""
+  transportHost: "amqp://amq-router-service-name.amq-namespace.svc.cluster.local"
+
+----
+
+You can create a Hardware Event Kind CR to observe hardware events and faults reported in the `hw-event-proxy` logs.
```

Comment:
I think we can delete this line and the preceding CR if we include the nodeSelector detail in the CR above.